src/gsy_e/gsy_e_core/simulation.py

```diff
@@ -588,7 +588,7 @@ def _execute_simulation(
 
                 # reset tick_resume after possible resume
                 tick_resume = 0
-                log.trace("Tick %s of %s in slot %s (%.1f%)", tick_no + 1,
+                log.trace("Tick %s of %s in slot %s (%.1f)", tick_no + 1,
```

Comment:
How come ? I think this log really intends to have a percentage value. Might it be that there was an exception raised and we had not seen it due to the trace level ?